components/crud-web-apps/jupyter/manifests/base/cluster-role.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cluster-role
+rules:
+- apiGroups:
```

Comment:
I think this file will need updating (or the file with ClusterRole jupyter-web-app-cluster-role) as the `crud-wb-apps/jupyter` also needs the permission:

```
- apiGroups:
  - ""
  resources:
  - nodes
  verbs:
  - get
  - list
  - watch
 ```

Reply:
Yeap, it may be out of date! But for this PR, I would suggest to leave it out of scope and focus on simply transferring back what kubeflow/manifests already has.